File: VelibStations_iOS/VelibStations/Location/ILocationManager.swift
//
//  ILocationManager.swift
//  VelibStations
//
//  Created by Safi on 01/10/2019.
//

import Foundation
import CommonKit
import CoreLocation

protocol ILocationManager : IAddOn {
    func requestUpdates()
    func cancelUpdates()
    func getLocation() -> CLLocation?
}

File: VelibStations_Android/common/src/commonMain/kotlin/com/muhammedsafiulazam/common/location/ILocationManager.kt
package com.muhammedsafiulazam.common.location

import com.muhammedsafiulazam.common.addon.IAddOn
import com.muhammedsafiulazam.common.model.Location

interface ILocationManager : IAddOn {
    fun requestUpdates()
    fun cancelUpdates()
    fun getLocation() : Location?
}